cli/doc/parser.rs

```diff
@@ -184,6 +184,14 @@ impl DocParser {
   pub async fn parse(&self, file_name: &str) -> Result<Vec<DocNode>, ErrBox> {
     let source_code = self.loader.load_source_code(file_name).await?;
 
+    self.parse_source(file_name, source_code.as_str())
+  }
+
+  pub fn parse_source(
+    &self,
+    file_name: &str,
+    source_code: &str,
+  ) -> Result<Vec<DocNode>, ErrBox> {
```

Comment:
If I understand correctly this method is needed only to parse given source code without using loader, right?

Reply:
Yeah, this was satisfied previously by the old `parse()` signature before reexport support.